crates/typst-library/src/foundations/array.rs

```diff
@@ -810,15 +788,15 @@ impl Array {
             .collect()
     }
 
-    /// Return a sorted version of this array, optionally by a given key
-    /// function. The sorting algorithm used is stable.
+    /// 配列のソート（並び替え）されたバージョンを返します。オプションとして、キー関数による
+    /// ソートも可能です。使用されるソートアルゴリズムでは、同順位要素の前後関係は
+    /// 変化しません（安定）。
```

Comment:
補足は非常にありがたいです。一方で`（安定）`という補足は「安定ソート」を知らない人が混乱してしまうため、「安定ソート」と言ってしまった方がよいかと思います。
```suggestion
    /// 配列のソート（並び替え）されたバージョンを返します。オプションとして、キー関数による
    /// ソートも可能です。使用されるソートアルゴリズムは安定ソートであり、同順位要素の前後関係は
    /// 変化しません。
```